### ISSUE TITLE
Protocol Error Warning

### ISSUE THREAD

**Issue by NullDev** [EpticMC/Bungee-AdvancedPlusOne#2]
The issue https://github.com/EpticMC/Bungee-AdvancedPlusOne/issues/1 has been fixed, but as a result the In-Game server list displays a protocol error. 
Connection to the server is still possible and the plugin works as expected. 

![issue_2](https://user-images.githubusercontent.com/22935000/30211841-c88faf1e-94a2-11e7-9456-0d159b77af86.jpg)

Causing code:

Probably the custom protocol handler around <a href="https://github.com/EpticMC/Bungee-AdvancedPlusOne/blob/master/src/org/nulldev/AdvancedPlusOne/Main.java#L58-L68">Line 58 to Line 68</a> in <a href="https://github.com/EpticMC/Bungee-AdvancedPlusOne/blob/master/src/org/nulldev/AdvancedPlusOne/Main.java">Main.java</a> but I haven't tested yet.

```Java
public void setProt(ServerPing response) {
    if (response != null) {
        this.version = response.getVersion();
        String max = Integer.toString(response.getPlayers().getMax());
        String online = Integer.toString(response.getPlayers().getOnline());
        if (online == max) this.version.setName((Object)ChatColor.GRAY + online + "/" + max + 1);
        else this.version.setName((Object)ChatColor.GRAY + online + "/" + max);
        this.version.setProtocol(999);
        response.setVersion(this.version);
    }
}
```

Needs investigation.

**Comment by ghost** [comment]
`this.version.setProtocol(999);`

this looks a bit off

**Comment by NullDev** [comment]
True. I just looked at the bungee API documentation again and it seems like this does indeed set the required client version. I don't know why its there. Maybe a left over from the experiments with https://github.com/EpticMC/Bungee-AdvancedPlusOne/issues/1

I compiled a version without setting the protocol. Ill let you know if it worked. I am not sure if the bungee proxy actually needs a specification on this one. If not, the function probably just overrides a default value.
